deployment/ansible/roles/model-my-watershed.geotrellis/templates/nginx-mmw-geotrellis.conf.j2

```diff
@@ -0,0 +1,23 @@
+server {
+  listen *:80;
+  server_name _;
+
+  access_log /var/log/nginx/mmw-geotrellis.access.log logstash_json;
+
+  {% if ['packer'] | is_in(group_names) -%}
+  location /version.txt {
+    alias /srv/version.txt;
+  }
+  {% endif %}
+
+  location / {
+    {% if ['packer'] | is_in(group_names) -%}
+    expires max;
```

Comment:
I'm not sure we'll want any HTTP caching on these requests. Are duplicate jobs being cached further up in the stack somewhere?